Jenkinsfile

```diff
@@ -119,16 +126,16 @@ helpers.rootLinuxNode(env, {
                                         "KEYBASE_SERVER_URI=http://${kbwebNodePrivateIP}:3000",
                                         "KEYBASE_PUSH_SERVER_URI=fmprpc://${kbwebNodePrivateIP}:9911",
                                     ]) {
-                                        testNixGo("Linux")
+                                        if (hasGoChanges) {
+                                            testNixGo("Linux")
+                                        }
                                     }},
                                     test_linux_js: { withEnv([
                                         "PATH=${env.HOME}/.node/bin:${env.PATH}",
                                         "NODE_PATH=${env.HOME}/.node/lib/node_modules:${env.NODE_PATH}",
                                     ]) {
                                         dir("shared") {
-                                            stage("JS tests") {
```

Comment:
Why remove the stage directive? Helps see where things are in the pipeline viewer